src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = price * days;
+
+    return result - 50;
+  } else if (days >= 3) {
```

Comment:
Consider using a constant for the number of days required for the $20 discount and the discount amount itself. This will help avoid hardcoding and improve readability. Refer to checklist item 2.